newIDE/app/src/ProjectManager/index.js

```diff
@@ -738,6 +760,27 @@ export default class ProjectManager extends React.Component<Props, State> {
                 )
             }
           />
+          {!!this.state.layoutVariablesDialogOpen && (
+            <VariablesEditorDialog
```

Comment:
Can we modify SceneEditor so that it's using this, rather than having the same implementation of VariablesEditorDialog used twice?

Reply:
The SceneEditor takes in other props that the project manager cannot provide at the moment (would have to rewrite the same implementation for variables/functions it takes as props).

Reply:
I mean SceneEditor has the same thing inside it:

https://github.com/4ian/GDevelop/blob/f1730c239dbaad98b3708d3e94d2586c597e08bb/newIDE/app/src/SceneEditor/index.js#L1235-L1238

This seems like an unecessary duplication of the this. How can you modify SceneEditor so that there is only one VariablesEditorDialog used for this specific case (i.e: the case of modifying scene variables)?